index.bs

```diff
@@ -6438,6 +6445,90 @@ requests will be affected.
 
 </div>
 
+#### The network.setCacheMode Command ####  {#command-network-setCacheMode}
+
+The <dfn export for=commands>network.removeSetCacheMode</dfn> command overrides
+the network cache behaviour for certain requests.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      network.SetCacheMode = (
+        method: "network.setCacheMode",
+        params: network.SetCacheModeParameters
+      )
+
+      network.SetCacheModeParameters = {
+        mode: "no-store" / null,
+        ? contexts: [+browsingContext.BrowsingContext]
+      }
+    </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl">
+      EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The <dfn export>WebDriver BiDi update request cache mode</dfn> steps given
```

Comment:
I assume we have to land these changes first and afterward the fetch ones?

Reply:
Yes.

Reply:
Oh and two more questions (which might also answer the fetch spec but maybe you could tell right away):

1. This also affects normal navigation (including reload)?
2. What if no network interception is enabled? It has to work then as well, right?

Reply:
Yes, this should affect all network requests, including reloads, whether or not network interception is enabled.  

I think we might want to also allow setting the cache mode when intercepting at the `beforeRequestSent` stage, in case you really only care about certain requests, but we can handle that in a separate PR.

Reply:
That is a good idea, could you at least file an issue for that? Thanks.